docker/gaffer-integration-tests/docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Update store properties files to point to the location of the Accumulo store to test against:
+cd ~/store-implementation/accumulo-store/src/test/resources/
+sed -i '16 a gaffer.store.properties.class=uk.gov.gchq.gaffer.accumulostore.AccumuloProperties' store.properties
+sed -i 's/localhost/localhost:58630/g' store.properties
```

Comment:
Rather than hardcoding the values, it would be useful to copy a store.properties file into the file system somewhere and replace the values in the src/test/resources with ones from that file. This would mean that we could run the tests in Kubernetes too using a config map instead of a volume mount.